cogs/obliterate.py

```diff
@@ -334,6 +397,17 @@ async def apply(self, interaction: discord.Interaction):
             return
         await interaction.response.send_modal(AccountInfoModal(self.bot))
 
+    @app_commands.command()
+    @app_commands.guilds(discord.Object(id=config['obliterate_guild_id']), discord.Object(id=config['test_guild_id']))
+    async def appreciate(self, *appreciate_players, interaction: discord.Interaction):
```

Comment:
Would this work to allow people to tag multiple players? (I'm sure i've seen people giving compliments to multiple people in one post in Cozy before)

Reply:
I'm not sure if that would be possible with a slash command, but it would certainly rule out adding autocomplete. It would also make getting the right user(s) to mention significantly more complex. The system used in the cozy server is very much flawed, I would just stick to one user per command for now